Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/IdentityProvidingClients/IdentityProvidingSSOClient.swift

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
The protocol for SSO client, used for dependency inversion. The concrete struct that conforms to this protocol is code generated at `Sources/Core/AWSSDKIdentity/Sources/InternalAWSSSO/IdentityProvidingSSOClient.swift`.